schemas.yaml

```diff
@@ -110,6 +110,14 @@ schemas:
     "$ref": schemas/job_preview.yaml#/job_preview
   job_locale:
     "$ref": schemas/job_locale.yaml#/job_locale
+  job_template:
```

Comment:
how come we were missing all these?

Reply:
honestly i don't know either. running `npx openapi-generator-cli validate -i tmp/compiled.yaml` will find all of these issues but it also finds a few more deprecated things i've not been able to fix so far